### DISCUSSION TITLE
Difficulty Changing App Name and Icon in Task Manager

### DISCUSSION THREAD

**Discussion by VitorAndrey** [alex8088/electron-vite#400] [Q&A]
Hello everyone,

I hope this question isn't too trivial, and I appreciate any help you can provide.

I'm currently facing challenges in changing the name and icon of my app in the task manager. I've successfully updated the app icon in the taskbar, tray, and window, but these changes don't seem to reflect in the task manager.

Additionally, I'm seeking more guidance on the "build" directory, as it's the only place where the default icon still exists. Can I safely delete this directory and create a new one with the updated icons? What exactly is the purpose of the "build" directory?

I'm also curious if there's a need to clear any build caches before running the app to ensure the icon change takes effect in the task manager.

Thank you in advance for any insights or advice on resolving these issues.

![image](https://github.com/alex8088/electron-vite/assets/114686823/0d75181d-13bb-4f48-b55a-b0fa91073c77)
![image](https://github.com/alex8088/electron-vite/assets/114686823/c6ebd7b4-e073-46c6-974f-b51637d26f69)

**Comment by alex8088** [comment]
The icon will be replaced after packaging

**Reply by VitorAndrey** [reply]
Thanx!
